### DISCUSSION TITLE
Unexpected token error when running Appwrite CLI

### DISCUSSION THREAD

**Discussion by Mathiaszero** [appwrite/appwrite#4633] [Q&A] [answered]
### 👟 Reproduction steps

1. Install Appwrite CLI by using command "npm install -g appwrite-cli" on Konsole
2. Run "appwrite -v" or "appwrite" on Konsole

### 👍 Expected behavior

There should not be any errors after using the command "appwrite" or "appwrite -v"

### 👎 Actual Behavior

The error outputs "usr/local/lib/node_modules/appwrite-cli/lib/parser.js:137" See screen capture:

![image](https://user-images.githubusercontent.com/55817193/199810011-8f2af7b3-b218-4b35-814d-cfb11d29b5f6.png)

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Kubuntu 22.04

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@Mathiaszero thanks for opening this issue! 🙏🏼 What version of node do you have? You may have an old version that requires updating.

**Comment by Mathiaszero** [comment]
> @Mathiaszero thanks for opening this issue! 🙏🏼 What version of node do you have? You may have an old version that requires updating.

Thanks for response.  The version I have is 12.22.9

**Comment by stnguyen90** [comment, marked as answer]
@Mathiaszero would you please try with Node v14?

**Comment by Mathiaszero** [comment]
@stnguyen90 Yes, and I appreciate your help. Just used "nvm install 14" and redid the command. Case closed!
